api/src/legacy_soap_api/legacy_soap_api_config.py

```diff
@@ -87,6 +90,9 @@ class SOAPOperationConfig:
     # namespaces or namespace prefixes.
     namespace_keymap: dict[str, str | None] = field(default_factory=dict)
 
+    def compare_endpoints(self) -> bool:
+        return self.request_operation_name in CompareRequestOperationName
```

Comment:
I was thinking we'd add this to the configuration below that just the `GetOpportunityListRequest` is configured for at the moment, no enum at all.